[2022-02-02 21:00:53,184] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-02-02 21:00:53,205] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-02-02 21:00:53,207] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:00:53,208] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-02-02 21:00:53,209] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:00:53,225] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-01 00:00:00+00:00
[2022-02-02 21:00:53,233] {standard_task_runner.py:52} INFO - Started process 16051 to run task
[2022-02-02 21:00:53,238] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'local_to_gcs_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '415', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpo004lmxe', '--error-file', '/tmp/tmpkw00da1d']
[2022-02-02 21:00:53,240] {standard_task_runner.py:77} INFO - Job 415: Subtask local_to_gcs_task
[2022-02-02 21:00:53,324] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.local_to_gcs_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host 700aeb16bae0
[2022-02-02 21:00:53,367] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-02 21:00:53,408] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-02-02 21:01:05,234] {python.py:175} INFO - Done. Returned value was: None
[2022-02-02 21:01:05,263] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=local_to_gcs_task, execution_date=20190101T000000, start_date=20220202T210053, end_date=20220202T210105
[2022-02-02 21:01:05,326] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 21:01:05,370] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
